ui/Screen/Onboarding/EnterName.svelte

```diff
@@ -73,7 +73,7 @@
     </p>
     <Input.Text
       autofocus={true}
-      placeholder="Enter a name"
+      placeholder="Enter a display name (i.e. coolprogrammer3000)"
```

Comment:
```suggestion
      placeholder="Enter a display name (e.g. coolprogrammer3000)"
```

I think that `i.e.` stands for "that is, ..."

Reply:
Good catch!